saleor/dashboard/product/forms.py

```diff
@@ -55,12 +63,21 @@ class Meta:
         model = ProductClass
         exclude = []
         labels = {
+            'name': pgettext_lazy(
+                'Product class form label',
+                'Product class name'),
+            'has_variants': pgettext_lazy(
+                'Product class form label',
+                'Check if product has more than one variant'),
```

Comment:
This label doesn't make much sense. This field is not about checking if a product has attributes, but about enabling them (showing the UI to add them), so maybe "Enable variants"? I'm not sure though.

Reply:
"Enable variants" sounds ok for me, but I'm open for discussion.